docs/reference/contributing/guidelines/workflow.md

```diff
@@ -36,31 +44,35 @@ Please see the [code contributions chapter](/docs/v5.7/reference/guidelines.html
-	1. Capitalize the subject line.
-	1. Do not end the subject line with a period.
-	1. Use the imperative mood in the subject line.
-	1. Wrap the body at 72 characters.
-	1. Use the body to explain _what_ and _why_ vs _how_.
+    1. Separate subject from body with a blank line.
+    1. Limit the subject line to 72 characters (note that this is a deviation from Beam's standard).
+    1. Capitalize the subject line.
+    1. Do not end the subject line with a period.
+    1. Use the imperative mood in the subject line.
+    1. Wrap the body at 72 characters.
+    1. Use the body to explain _what_ and _why_ vs _how_.
 - Because we use GitHub and explicit CLAs, special commit tags that other projects may use, such as “Reviewed-by”, or “Signed-off-by”, are redundant and should be omitted. GitHub tracks who reviewed what and when, and our stack of signed CLAs shows us who has agreed to our development contribution agreement.
 - Prefixing your commit message with a domain is acceptable, and we recommend doing so when it makes sense. However, prefixing one's domain with the name of the repo is not useful. For example, making a commit entitled "mbed-drivers: Fix doppelwidget frobulation" to the `mbed-drivers` repo is not acceptable because it is already understood that the commit applies to `mbed-drivers`. Renaming the commit to "doppelwidget: Fix frobulation" would be better, if we presume that "doppelwidget" is a meaningful domain for changes, because it communicates that the change applies to the doppelwidget area of `mbed-drivers`.
 - All new features and enhancements require documentation, tests and user guides for us to accept them. Please link each pull request to all relevant documentation and testing pull requests.
+- a pull request shall fix a bug, add a feature or refactor but not a mixture
 
 #### Mbed OS maintainers
 
-The maintainers are a small group of Mbed OS engineers who are responsible for the Mbed OS codebase. Their primary role is to progress contributions, both internal and external, from the initial pull request state through to released code. They:
+The maintainers are a small group of Mbed OS engineers who are responsible for the Mbed OS codebase. Their primary role is to progress contributions, both internal and external, from the initial pull request state through to released code. 
+
+Responsibilities:
 
 1. Check for CLA compliance.
-2. Ensure the relevant stakeholders review pull requests.
-3. Guide contributors both technically and procedurally.
-4. Run pull requests through the CI systems.
-5. Merge pull requests into the requested branches.
-6. Make periodic patch and feature releases.
+1. Ensure the relevant stakeholders review pull requests.
+1. Guide contributors both technically and procedurally.
+1. Run pull requests through the CI systems.
+1. Put label version.
+1. Merge pull requests into the requested branches.
+1. Make periodic patch and feature releases.
```

Comment:
Again you seem to have lost the numbering ?

Reply:
markdown does it automatically, it is always a probelm when adding /changing - all numbers affected. this way can do without editing everything

Reply:
Do your changes render correctly when you view with a markdown viewer now ?

Reply:
Yes, please view it to verify

Reply:
Numbers look fine when rendered: https://github.com/0xc0170/Handbook/blob/d65fe18cdb89888f01733f5c1b186830ea050dc1/docs/reference/contributing/guidelines/workflow.md